tests/comment.test.js

```diff
@@ -0,0 +1,86 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import dataGenerator from './dataGenerator';
+import app from '../index';
+import utils from './utils';
+
+chai.use(chaiHttp);
+chai.should();
+
+let token = '';
+
+describe('Comments', () => {
+  // get the token
+  before((done) => {
+    utils
+      .getUserToken()
+      .then((res) => {
+        token = res.body.user.token;
```

Comment:
better if also can apply this suggestion of hound to use destructuring.

Reply:
The variable is defined in the upper scope so that we can access it everywhere. And I have not been able to destructure into a defined variable so far! If you know how you can help me.